src/components/Hero.js

```diff
@@ -2,24 +2,24 @@ import React from 'react';
 
 const Hero = () => {
   return (
-    <section className='text-gray-700 body-font'>
+    <section className=' font-body bg-secondary-blue h-screen'>
```

Comment:
The Hero section has a gab at the bottom. Either we need to fill it with something or reduce the size.

Reply:
may be we can reduce the Viewport size to 80vh.

Reply:
try viewport, could be helpful